test/acceptance/tests/fixtures/crds/servicerouter.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: consul.hashicorp.com/v1alpha1
```

Comment:
ServiceRouter requires its protocol be defined as http so we need a ServiceDefaults too